addon/components/rdf-input-fields/concept-scheme-selector.js

```diff
@@ -33,17 +37,40 @@ export default class RdfInputFieldsConceptSchemeSelectorComponent extends InputF
   loadOptions() {
     const metaGraph = this.args.graphs.metaGraph;
     const fieldOptions = this.args.field.options;
+    let conceptScheme = null;
 
     if (!hasValidFieldOptions(this.args.field, ['conceptScheme'])) {
-      return;
-    }
+      const scheme = this.args.formStore.any(
+        this.args.field.uri,
+        FORM_OPTION('conceptScheme'),
+        undefined,
+        this.args.graphs.formGraph
+      );
+      if (!scheme) {
+        return;
+      }
 
-    const conceptScheme = new namedNode(fieldOptions.conceptScheme);
+      conceptScheme = scheme;
+    } else {
+      conceptScheme = new namedNode(fieldOptions.conceptScheme);
+    }
 
     /**
      * NOTE: Most forms are now implemented to have a default "true" behavior
      */
-    if (fieldOptions.searchEnabled !== undefined) {
+    if (!hasValidFieldOptions(this.args.field, ['searchEnabled'])) {
+      const isEnabled = this.args.formStore.any(
+        this.args.field.uri,
+        FORM_OPTION('searchEnabled'),
+        undefined,
+        this.args.graphs.formGraph
+      );
+      if (!isEnabled) {
+        return;
+      }
+
+      this.searchEnabled = isEnabled == '1' ? true : false;
+    } else {
       this.searchEnabled = fieldOptions.searchEnabled;
     }
```

Comment:
This is a comparison of checking it in the component itself or using it through the base input component.
```js
  getOptionPredicates() {
    return {
      conceptScheme: EXT('conceptScheme'),
    };
  }

  loadOptions() {
    const metaGraph = this.args.graphs.metaGraph;
    const conceptScheme = this.findFieldOption('conceptScheme', 'node');

    if (!conceptScheme) {
      return;
    }

Reply:
there are also more components where we need to add the new way of fetching options (by predicate) right? this PR is just an example for now?

Reply:
@Rahien Yes, it only an example so we can compare the implementations and choose which option to go for.